tests/e2e/test_main.py

```diff
@@ -237,13 +244,62 @@ def test_image_build_custom_build_args(cli_runner: CLIRunner) -> None:
     assert f"arg-another-{tag}" in result.stdout
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="kaniko does not work on Windows")
+def test_image_build_env(cli_runner: CLIRunner) -> None:
```

Comment:
may we also consider adding test with `-v` / `--volume` passed? 
Or it will be redundant?

Reply:
@YevheniiSemendiak please take a look at the new test. If it's ok, please approve the PR and I'll merge it right away :)